src/component/todoList.js

```diff
@@ -0,0 +1,68 @@
+export default function TodoList(app) {
+	this.$todoList = document.querySelector("#todo-list");
+	this.todoItems = [];
+
+  this.setState = updatedTodoItems => {
+    this.todoItems = updatedTodoItems;
+    this.render(this.todoItems);
+  };
+
+  this.render = items => {
+    const template = items.map(todoItemTemplate);
+    this.$todoList.innerHTML = template.join("");
+  };
+
+	const todoItemTemplate = (item) => {
+    return `<li class=${item.status}>
+    <div class="view">
+      <input class="toggle" type="checkbox" ${item.status === "completed" ? "checked": ""}/>
+      <label class="label">${item.text}</label>
+      <button class="destroy"></button>
+    </div>
+    <input type="hidden" id="item-id" value="${item.id}"/>
+    <input class="edit" value="${item.text}" />
+  </li>`;
+	};
+
+	this.filterItems = status => {
+		if (status !== "all") {
+			this.todoItems = app.todoItems.filter(item => item.status === status);
+			this.render(this.todoItems);
+		}
+		else {
+			this.render(app.todoItems);
+		}
```

Comment:
`else` 문을 사용하지 않게도 만들어 볼 수 있겠네요 👍  
```suggestion
		if (status === "all") return this.render(app.todoItems)
		this.todoItems = app.todoItems.filter(item => item.status === status);
	        this.render(this.todoItems);
```